tls/s2n_kem.c

```diff
@@ -182,15 +169,15 @@ const struct s2n_iana_to_kem kem_mapping[3] = {
  * || ECC key share (variable bytes)
  * || size of PQ key share (2 bytes)
  * || PQ key share (variable bytes) */
-const struct s2n_kem_group s2n_secp256r1_sike_p434_r2 = {
+const struct s2n_kem_group s2n_secp256r1_sike_p434_r3 = {
         .name = "secp256r1_sike-p434-r2",
```

Comment:
I'm debating whether we should change the name or not. I think `secp256r1_sike-p434-r2r3` might be clearer that the KEM Group applies to both versions. Same goes for the variable name `s2n_secp256r1_sike_p434_r3`.

Reply:
Not sure if it sways your opinion on the matter at all, but I _meant_ to change the name string to `r3`.

Reply:
After some thought, I think I'm sticking with the opinion of renaming everything to just `r3`. My main reasoning is that while round 3 is compatible with r2, the implementation that we have in s2n is the round 3 implementation. I think `r2r3` may end up being more confusing than clarifying. Probably the best way to indicate the compatibility is with an explanatory comment.